molecule/default/test_index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
```

Comment:
Let me just say that I like this file.

Reply:
It might [look familiar](https://mirror.openshift.com/pub/openshift-v4/dependencies/rhcos/pre-release/latest/) to anyone trying to write automation around RH CoreOS :)